src/area-chart/area-chart-stacked.component.ts

```diff
@@ -356,8 +361,13 @@ export class AreaChartStackedComponent extends BaseChartComponent {
       domain.push(sum);
     }
 
-    const min = Math.min(0, ...domain);
-    const max = Math.max(this.yAxisMinScale, ...domain);
+    const min = this.yScaleMin
```

Comment:
Wrong value check, I think 
`this.yScaleMin !== undefined && this.yScaleMin !== null`
should be better ;)

if you use lodash, `!_.isNil(this.yScaleMin)`